src/System.Private.ServiceModel/tests/Common/Infrastructure/src/project.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "System.Net.Sockets" : "4.0.10-beta-*",
```

Comment:
If this test still needs System.Net.Sockets, follow the example @mconnew provided in src\System.Private.ServiceModel\src\project.json.  It moves into its own special area at the end.